rabbitmq/ci/start-docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+source utils/bash/ci.sh
+
+NAME='dd-test-rabbitmq'
+
+stop-docker $NAME
```

Comment:
That's a good question. That'w why I tried to make it a pretty esoteric name. Should we print it out that we're doing it?